### DISCUSSION TITLE
Organizations

### DISCUSSION THREAD

**Discussion by MrJhoker** [osTicket/osTicket#6142] [Q&A]
Hello, does anyone know how to automatically add users to organizations when they generate a ticket? They should be added automatically when the option "Automatically add members of " is checked but I can't get them added. I'm using version 1.16. Thank you

**Comment by ntozier** [comment]
This does not appear to be a bug report.  I would encourage you to seek your support experience on the forums:
https://forum.osticket.com/

I believe that option triggers are account creation.  If the users already exist then you would have to add them manually. [note: this is not a feature that I use.]

**Reply by MrJhoker** [reply]
Ok, I'll put it on the forum. Thanks anyway :D
